index.html

```diff
@@ -43,7 +43,7 @@ <h1 class="sr-only">Bikes for Refugees Scotland - Providing donated bikes and ac
 							<a class="nav-link" href="#">Contact Us</a>
 						</li>
 					</ul>
-					<button>Donate Now</button>
+					<button class = "donate-button">Donate Now</button>
```

Comment:
You probably haven't covered this yet but there are button classes in bootstrap which will style this button really nicely like in the example with rounded corners etc. Try adding the class `btn` to all the buttons if you want to see how it styles it. Here's a link with some of the other possible classes https://www.w3schools.com/bootstrap4/bootstrap_buttons.asp